react/features/aeternity/components/TipButton.js

```diff
@@ -174,10 +174,20 @@ class TipButton extends Component<Props, State> {
      * @returns {void}
      */
     _onChangeValue({ target: { value } }) {
-        const validationRegExp = /^\d*\.?\d*/;
+        const validationRegExp = /^\d+\.?\d*$/;
         const result = value.match(validationRegExp);
```

Comment:
if I'm not wrong, you can refact it with `destructuring assignment `
```suggestion
        const [ result ] = value.match(validationRegExp);
```

Reply:
or even maybe 
```suggestion
        const [ result = [] ] = value.match(validationRegExp);
```